spec/models/salt_event_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+describe SaltEvent do
+  describe ".process_next_event" do
+    let(:leftover_event) do
+      # not timed out
+      FactoryGirl.create(
+        :salt_event, worker_id: "this_worker", processed_at: nil,
+        taken_at: Time.current - (SaltEvent::PROCESS_TIMEOUT_MIN - 2).minutes
```

Comment:
Here I'd use `(SaltEvent::PROCESS_TIMEOUT_MIN - 2).minutes.ago`, is more idiomatic.